content_sync/pipelines/definitions/concourse/mass-build-sites.yml

```diff
@@ -113,6 +113,7 @@ jobs:
       GIT_KEY: ((git-private-key-var))
       SITEMAP_DOMAIN: ((sitemap-domain))
       PREFIX: ((prefix))
+      OCW_HUGO_THEMES_SENTRY_DSN: ((ocw-hugo-themes-sentry-dsn))
```

Comment:
It's good to be specific when it comes to the Django setting as there could be other Sentry DSN's potentially in the `ocw-studio` environment, but in the pipelines I think we could just call it `SENTRY_DSN` for simplicity.